addons/binding/org.openhab.binding.harmonyhub/src/main/java/org/openhab/binding/harmonyhub/handler/HubStatusListener.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.harmonyhub.handler;
+
+import org.eclipse.smarthome.core.thing.ThingStatus;
+
+/**
+ * the {@link HarmonyDeviceHandler} interface is for classes wishing to register
+ * to be called back when a HarmonyHub status changes
+ *
+ * @author Dan Cunningham - Initial contribution
+ *
+ */
+public interface HubStatusListener {
```

Comment:
Fortunately a scan is very cheap, all it does is ask the bridge/hub for a list of devices.